pandas/core/frame.py

```diff
@@ -5644,6 +5651,16 @@ class  max_speed
         2    lion  mammal       80.5
         3  monkey  mammal        NaN
 
+        Using the `names` parameter, it is possible to choose a name for
```

Comment:
need to add the names parmater on line 5617, e.g. in the doc-string proper. add a versionadded 1.4 label.